changelogs/unreleased/gh-9159-fix-memleak-in-sql-column-default-parser.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/sql
+
+* Fixed a memory leak in a column default parser (gh-9159).
```

Comment:
I think it is better to write `column default rule`.

Reply:
Fixed to `* Fixed a memory leak in a parser for column default rule (gh-9159).`